modules/core/src/main/java/org/locationtech/jts/geomgraph/EdgeRing.java

```diff
@@ -98,7 +98,7 @@ public void computeRing()
       coord[i] = (Coordinate) pts.get(i);
     }
     ring = geometryFactory.createLinearRing(coord);
-    isHole = Orientation.isCCW(ring.getCoordinates());
+    isHole = Orientation.isCCW(ring.getCoordinateSequence());
```

Comment:
Perhaps this should be a separate PR, since I don't think it is related to normalize issue?

Reply:
Agreed, my enthusiasm. Belongs to #188.